gsadjust/gui/tabs/drift.py

```diff
@@ -237,6 +247,9 @@ def __init__(self, parent):
         layout_main.addWidget(main_vsplitter_window)
         self.setLayout(layout_main)
 
+        # Call reset to initialize the table models.
+        self.reset()
+
     def reset(self):

```

Comment:
If you have a reset method like this, it's is usually good to re-use it for the setup during init (to avoid code duplication).